src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
+$day-size: 100px;
+$border-size: 1px;
+$gap-size: 1px;
+$padding: 10px;
+$calendar-width: calc(#{$day-size} * 7 + #{$gap-size} * 6 + #{$padding} * 2);
```

Comment:
There is a duplication of the calculation for `$calendar-width` here and on line 18. You could use the `$calendar-width` variable in the `.calendar` class width property to avoid repetition and potential errors if the calculation needs to be updated.